clients/python/setup.py

```diff
@@ -3,6 +3,6 @@
 setup(
     name="scale-llm-engine",
     python_requires=">=3.7",
-    version="0.0.0.beta25",
+    version="0.0.0.beta26",
```

Comment:
could we make the version bump change in pyproject.toml also?

Reply:
ah thanks for the catch